packages/cli-utils/src/ts/virtualCode/vue.ts

```diff
@@ -0,0 +1,35 @@
+import type { VirtualCode } from '@vue/language-core';
+import { forEachEmbeddedCode, getDefaultVueLanguagePlugins } from '@vue/language-core';
+import * as vueCompilerDOM from '@vue/compiler-dom';
+import * as vue from '@vue/language-core';
+
+import type { CreateVirtualCode } from './types';
+
+const VueVirtualCode =
+  vue.VueGeneratedCode || ((vue as any).VirtualCode as typeof vue.VueGeneratedCode);
```

Comment:
This export was renamed on the `main` branch of `@vuejs/language-tools`, so both names will have to be checked here to anticipate their next release: https://github.com/vuejs/language-tools/commit/f46634c7e5a330032b40d224cf72dd1c0b6640d5